### ISSUE TITLE
Feature: Show current charge status in XMPP status

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#92]
```
Would be nice if gtalksms shows the current charge status in XMPP status:
charging (USB)
charging (AC)
discharging

```

Original issue reported on code.google.com by `fschm...@gmail.com` on 9 Feb 2011 at 1:58

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 3 Mar 2011 at 4:24
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.0
```

Original comment by `Florent....@gmail.com` on 8 Mar 2011 at 7:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
